_posts/2022-02-15-uncovering-bad-commits.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: Uncovering Bad Commits
```

Comment:
Title suggestion: Finding a needle in a haystack - Uncovering bad commits with git bisect

And suggestion for an image to include in the post - https://unsplash.com/s/photos/needle-haystack

Reply:
Title was reworded to include "needed in a haystack". Also grabbed an image.
<img width="779" alt="image" src="https://user-images.githubusercontent.com/16843/154359784-6045609a-c3d8-40df-89bb-58f1b599c075.png">